src/s3.rs

```diff
@@ -11275,7 +11275,9 @@ impl<P, D> S3Client<P, D> where P: ProvideAwsCredentials, D: DispatchSignedReque
             None => request.set_content_type("binary/octet-stream".to_string())
         };
 
+        // s3.cn-north-1.amazonaws.com.cn for cn-north-1
         let hostname = (&input.bucket).to_string() + ".s3.amazonaws.com";
+        //let hostname = self.hostname(Some(&input.bucket));
```

Comment:
This is the merge conflict and I believe it needs to be set to this commented out line, not the line above it.